tests/fromRdf-manifest.jsonld

```diff
@@ -379,6 +379,22 @@
       "input": "fromRdf/js11-in.nq",
       "expect": "fromRdf/js11-out.jsonld",
       "option": {"specVersion": "json-ld-1.1"}
+    }, {
+      "@id": "#tjs12",
+      "@type": ["jld:PositiveEvaluationTest", "jld:FromRDFTest"],
+      "name": "JSON literals (strings)",
+      "purpose": "Tests creating property with multiple rdf:type rdf:JSON to a JSON literal (null).",
```

Comment:
the 'name' is not accurate, and I don't understand the sentence under 'purpose' (using "`rdf:type`" to talk about the datatype is very confusing).
```suggestion
      "name": "Multiple JSON literals",
      "purpose": "Tests creating property with multiple literal values of datatype rdf:JSON",
```